website/docs/reference/widgets/table/README.md

```diff
@@ -72,171 +83,144 @@ Controls how overflowing contents of the column are handled. When turned on, the
-| **isVisible**          | This property indicates whether the widget is visible or not.                                                                | `{{<table_name>.isVisible}}`           |
-| **sortOrder**          | This property helps you sort values.                                                                                         | `{{<table_name>.sortOrder}}`           |
-| **tableData**          | This property gets all the table data in JSON format.                                                                        | `{{<table_name>.tableData}}`           |
-| **selectedRowIndex**   | It gives the index of the row selected by the user. Not applicable when multiple rows are selected.                          | `{{<table_name>.selectedRowIndex}}`    |
-| **selectedRowIndices** | It gives you an array of the index of the rows selected by the user. Not applicable when multi-row selection is turned off.    | `{{<table_name>.selectedRowIndices`\}} |
-| **filteredTableData**  | Contains the data of the rows left after applying any selected filters, sort rule, or search terms.                  | `{{<table_name>.filteredTableData}}`   |
-| **pageNo**             | Contains the current page number that the user is on. APIs can use it for pagination                                         | `{{<table_name>.pageNo}}`              |
-| **pageSize**           | Contains the number of rows that can fit inside a page of the table. Changes along with the height & row height of the table | `{{<table_name>.pageSize}}`            |
-| **searchText**         | Contains the search text entered by the user in the Table                                                                    | `{{<table_name>.searchText}}`          |
+#### Client side search
 
-For the following examples of binding properties, consider a table called `Table_1` with the table data given below:
+Client side search controls whether the user can search the entire data set for their query, or only what's currently shown in the table widget.
 
-```
-[
-  {
-    "step": "#1",
-    "task": "Drop a table",
-    "status": "✅",
-    "action": ""
-  },
-  {
-    "step": "#2",
-    "task": "Create a query fetch_users with the Mock DB",
-    "status": "--",
-    "action": ""
-  },
-  {
-    "step": "#3",
-    "task": "Bind the query using => fetch_users.data",
-    "status": "--",
-    "action": ""
-  }
-]
-```
+#### Column sorting
 
-And this is how the data looks in the table widget:
+Toggles whether table columns are sort-able. When turned on, users may click column headers to sort the table rows by that column's value. This setting only applies while the app is in View mode (where the app is deployed).
 
-![](/img/table\_sample\_data.png)
+<VideoEmbed host="youtube" videoId="hmi7BaF3jFI" title="Table | Column Sorting" caption="Use this setting to enable sorting rows by column value."/>
 
-**selectedRow**
+#### Visible
```

Comment:
i think we don't need to explain this in detail. Whatever is mentioned in the Properties table is enough